microsoft-edge/web-platform/release-notes/134.md

```diff
@@ -210,6 +202,16 @@ The method returns a `FetchLaterResult` that contains a boolean field.  The valu
 See [Window: fetch() method](https://developer.mozilla.org/docs/Web/API/Window/fetch) at MDN.
 
 
+<!-- ---------- -->
+###### Support Web Locks API in Shared Storage
+
+This feature makes concurrent execution of shared storage worklets more reliable by integrating the Web Locks API into Shared Storage.
+
+* `navigator.locks.request()` is available to worklet environments.
+* All modifier methods support the option `{ withLock: <resource>}`.
```

Comment:
```suggestion
* All modifier methods support the option `{withLock: <resource>}`.
```

Reply:
result:
```
* The `navigator.locks.request()` method is available to worklet environments.
* All modifier methods support the option `{withLock: <resource>}`.
* The `sharedStorage.batchUpdate(methods, options)` method allows multiple modifier methods to be executed atomically, by using the `withLock` option.  This enables a website to maintain consistency while updating data across multiple storage keys.
```
@captainbrosset 
* added 'method' descriptor per general wording pattern in 134.md: 
* Not just `foo` but "the `foo` bar"